index.html

```diff
@@ -693,14 +706,93 @@ <h4>Proof Verification (ecdsa-2019)</h4>
         </section>
       </section>
     </section>
-    <section>
+    <section class="informative">
       <h2>Security Considerations</h2>
-      <p>
-The following section describes security considerations that developers
-implementing this specification should be aware of in order to create secure
-software.
+      <p>The security (integrity/authenticity) of a verifiable credential 
```

Comment:
The style of the rest of the doc is to have `<p>` and `<li>` start tags on their own line.

Reply:
Fixed.